spec/performance/injection_helper_spec.rb

```diff
@@ -6,8 +6,8 @@
   let(:oc) { create(:simple_order_cycle) }
   let(:relative_supplier) { create(:supplier_enterprise) }
   let(:relative_distributor) { create(:distributor_enterprise) }
-
   before do
+    pending("#9947")
```

Comment:
I think we never used to run performance specs on CI because they can take time and CI's performance is too unreliable.

Reply:
Yes, since they are broken already, let's remove them. We can also remove the used helpers.

Reply:
The only helper I've found to be referred in performance specs is `spec/support/file_helper.rb` but this one is used on other specs, so I think we can't remove it.